src/utils.c

```diff
@@ -1378,3 +1385,96 @@ ts_table_has_tuples(Oid table_relid, LOCKMODE lockmode)
 	table_close(rel, lockmode);
 	return hastuples;
 }
+
+/*
+ * This is copied from PostgreSQL 16.0 since versions before 16.0 does not
+ * support lists for privileges.
+ */
+static AclMode
+convert_any_priv_string(text *priv_type_text, const priv_map *privileges)
```

Comment:
```suggestion
ts_convert_any_priv_string(text *priv_type_text, const priv_map *privileges)
```

I suggest to prefix this function to make it clear that we call our version of the function and not the PostgreSQL version.

Reply:
No strong opinion, but it is a static function (as is the original, so there is no risk of using the wrong one) and we traditionally only add prefix to functions with external linkage.